install/kubernetes/helm/istio/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: istio
-version: 1.1.0
+version: 1.1.1
```

Comment:
we have automation to change helm chart versioning.  Please  rever this change.

TIA :)